datanode/docker/Dockerfile_reverseproxy

```diff
@@ -0,0 +1,46 @@
+# Copyright 2018 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Definition for Docker image that proxies traffic to the storage API service.
+
+# To build an image from this Dockerfile:
+# docker image build -f Dockerfile_reverseproxy -t interussplatform/reverse_proxy .
+
+# To run this image:
+# docker run \
+#     -e STORAGE_API_SERVER="storage_api_ip" -e STORAGE_API_PORT="8121" \
+#     -p 8121:8121 -p 8120:8120 -d interussplatform/reverse_proxy
+
+# Required environment variables:
+#   STORAGE_API_SERVER: Address of the storage API service.
+#   STORAGE_API_PORT: Port on which the storage API service is listening.
+
+# Optional environment variables:
+#   INTERUSS_API_PORT_HTTP: Port on which the InterUSS API will be served via HTTP (default 8121).
+#   INTERUSS_API_PORT_HTTPS: Port on which the InterUSS API will be served via HTTPS (default 8120).
```

Comment:
Can we reverse these? 8120 should be http, 8121 for https. This aligns to lower number, lower functionality, and maps to the current deployment.

Also, how would you turn OFF http? not saying it is a requirement, but if someone wanted to run only a SSL server, is it possible only with envoirnmental variables?

Reply:
Yep, will reverse.

There isn't a way to turn off HTTP in this PR, but I'm intending to add an option to have nginx issue 301 redirects on the HTTP endpoint.  It's not on this PR because this PR is already pretty big.